src/components/activities/patientDetailsActivity/PatientDetailsActivity.tsx

```diff
@@ -441,26 +337,79 @@ const PatientDetailsActivity: FunctionComponent<TProps> = ({
                   </div>
                 </div>
                 <div className="patientDetails__content">
-                  {userSection === "laboratory" ? (
-                    <div className="patientDetails__nested_content">
-                      <PatientDetailsContent
-                        title="Laboratory"
-                        content={PatientExams}
-                      />
-                    </div>
-                  ) : userSection === "admissions" ? (
-                    <div className="patientDetails__nested_content">
-                      <PatientDetailsContent
-                        title="Admissions"
-                        content={PatientAdmissions}
-                      />
-                    </div>
-                  ) : (
-                    <RouterTabs
-                      config={getRouteConfig()}
-                      defaultRoute={defaultRoute}
-                    />
-                  )}
+                  <div className={"patientDetails__nested_content"}>
+                    <Switch>
+                      <Route exact path={`${path}`}>
+                        <Redirect to={`${url}/summary`} />
+                      </Route>
+                      <Route path={`${path}/summary`}>
+                        <PatientDetailsContent
+                          title="Summary"
+                          content={PatientSummary}
+                        />
+                      </Route>
+                      <Route path={`${path}/admissions`}>
+                        {patient?.data?.status === PatientDTOStatusEnum.O ? (
+                          <Redirect to={`${url}/summary`} />
+                        ) : (
+                          <PatientDetailsContent
+                            title="Admissions"
+                            content={PatientAdmission}
+                          />
+                        )}
+                      </Route>
+                      <Route path={`${path}/visits`}>
+                        <PatientDetailsContent
+                          title="Visits"
+                          content={
+                            patient?.data?.status === PatientDTOStatusEnum.O
+                              ? PatientOPD
+                              : PatientVisit
```

Comment:
The right condition is inverse: if the patient is outpatient the app shows Visits instead shows OPD.